engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/K8sPodsResourceQuotaChecker.scala

```diff
@@ -0,0 +1,45 @@
+package pl.touk.nussknacker.k8s.manager
+
+import com.typesafe.scalalogging.LazyLogging
+import skuber.Resource.ResourceList
+import skuber.{ListResource, Resource, ResourceQuotaList}
+
+import scala.concurrent.Future
+import scala.language.reflectiveCalls
+
+object K8sPodsResourceQuotaChecker extends LazyLogging {
+
+  val podsResourceQuota = "pods"
+
+  def hasReachedQuotaLimit(oldDeploymentReplicasCount: Option[Int], quotas: ResourceQuotaList, replicasCount: Int): Future[Boolean] = {
```

Comment:
this method is synronous so why does it return Future?

Reply:
changed